openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -111,7 +116,16 @@ class LearningSequence(TimeStampedModel):
     learning_context = models.ForeignKey(
         LearningContext, on_delete=models.CASCADE, related_name='sequences'
     )
-    usage_key = UsageKeyField(max_length=255)
+    usage_key = UsageKeyField(max_length=255, unique=True)
+
+    # A URL-safe Base64-encoding of a blake2b hash of the usage key.
+    # For aesthetic use (eg, as a path parameter, to shorten URLs).
+    # This field is experimental. See TNL-8638 for more information.
+    usage_key_hash = models.CharField(
+        max_length=USAGE_KEY_HASH_LENGTH,
+        blank=True,
```

Comment:
We can't allow blanks, can we? And shouldn't there be a default value of `None` specified on the field?

Reply:
@ormsbee Sorry, I can never keep the ORM's null/blank/default semantics straight in my head.

I want the field start off as nullable (to permit the not-yet-backfilled state), with NULL as the initial value. I want to disallow empty values. Down the road, once this has stabilized and we've run a full backfill, we could make it non-nullable.

So, I would want  `null=True, default=None` in this PR, right?

Reply:
Yes, I think that's the right incantation.